master/security/calico-network-policy.md

```diff
@@ -0,0 +1,269 @@
+
+#### Write once, works everywhere
+No matter which cloud provider you use now, adopting Calico network policy means you write the policy once and it is portable. If you move to a different cloud provider, you don’t need to rewrite your Calico network policy. Calico network policy is a key feature to avoid cloud provider lock-in. 
+
+#### Works seamlessly with Kubernetes network policies
+You can use Calico network policy in addition to Kubernetes network policy, or exclusively. For example, you could allow developers to define Kubernetes network policy for their microservices. For broader and higher-level access controls that developers cannot override, you could allow only security or Ops teams to define Calico network policies.
+
+### Features
+
+The **Calico network policy API** supports the following features:
+- Policies can be applied to any kind of endpoint: pods/containers, VMs, and/or to host interfaces
+- Policies can define rules that apply to ingress, egress, or both
+- Policy rules support:
+  - **Actions**: allow, deny, log, pass 
+  - **Source and destination match criteria**:
+    - Ports: numbered, ports in a range, and Kubernetes named ports
+    - Protocols: TCP, UDP, ICMP, SCTP, UDPlite, ICMPv6, protocol numbers (1-255)
+    - HTTP attributes (if using Istio service mesh)
+    - ICMP attributes
+    - IP version (IPv4, IPv6)
+    - IP or CIDR
+    - Endpoint selectors (using label expression to select pods, VMs, host interfaces, and/or network sets)
+    - Namespace selectors
+    - Service account selectors
+- **Optional packet handling controls**: disable connection tracking, apply before DNAT, apply to forwarded traffic and/or locally terminated traffic
+
+### Concepts
+
+#### Endpoints
+Calico policies apply to **endpoints**. In Kubernetes, each pod is a Calico endpoint. However, Calico can support other kinds of endpoints. There are two types of Calico endpoints: **workload endpoints** (such as a Kubernetes pod or OpenStack VM) and **host endpoints** (an interface or group of interfaces on a host). 
+
+#### Namespaced and global network policies
+**Calico network policy** is a namespaced resource that applies to pods/containers/VMs in that namespace. 
+
+<pre>
+apiVersion: projectcalico.org/v3
+kind: <b>NetworkPolicy</b>
+metadata:
+  <b>name</b>: allow-tcp-6379
+  <b>namespace</b>: production
+</pre>
+{: .no-select-button}
+
+**Calico global network policy** is a non-namespaced resource and can be applied to any kind of endpoint (pods, VMs, host interfaces) independent of namespace. 
+
+<pre>
+apiVersion: projectcalico.org/v3
+kind: <b>GlobalNetworkPolicy</b>
+metadata:
+  <b>name</b>: allow-tcp-port-6379
+</pre>
+{: .no-select-button}
+
+**Calico network and global policies** are applied using calicoctl. Syntax is similar to Kubernetes, but there a few differences. For help, see [calicoctl user reference]({{site.baseurl}}/{{page.version}}/reference/calicoctl/).
```

Comment:
The way this sentence is structured implies that it's **Calico global policy** but that's not a real thing!
```suggestion
**Calico network policies** and **Calico global network policies** are applied using calicoctl. Syntax is similar to Kubernetes, but there a few differences. For help, see [calicoctl user reference]({{site.baseurl}}/{{page.version}}/reference/calicoctl/).
```

Reply:
Yes, please change to: "Calico network policies and Calico global network policies are applied..."

Reply:
Fixed.